src/ga4gh/vrs/dataproxy.py

```diff
@@ -88,6 +119,56 @@ def translate_sequence_identifier(self, identifier, namespace=None):
             nsd = namespace + ":"
             aliases = [a for a in aliases if a.startswith(nsd)]
         return aliases
+        
+    def derive_refget_accession(self, ac: str):
+        """Derive the refget accession from a public accession identifier
+
+        :param ac: public accession in simple or curie form from which to derive the refget accession
+        :return: Refget Accession if found
+        """
+
+        if ac == None:
```

Comment:
https://docs.astral.sh/ruff/rules/none-comparison/

```suggestion
        if ac is None:
```